mmdet/datasets/coco.py

```diff
@@ -416,6 +419,17 @@ def evaluate(self,
                 cocoEval.evaluate()
                 cocoEval.accumulate()
                 cocoEval.summarize()
+
```

Comment:
Does `text_eval` rely on `cocoEval.evaluate()` results? If no, why do you need to call it all the time?

Reply:
You are right. Will fix it